executor/adapter.go

```diff
@@ -718,6 +718,7 @@ func (a *ExecStmt) buildExecutor() (Executor, error) {
 		if err != nil {
 			return nil, err
 		}
+		a.Ctx.SetValue(sessionctx.QueryString, executorExec.stmt.Text())
```

Comment:
only DDL in Exec stmt should be added?

Reply:
Yeah, `QueryString` is used only for DDL.